nav2_route/graphs/geojson/aws_graph.geojson

```diff
@@ -0,0 +1,131 @@
+{
+"type": "FeatureCollection",
+"name": "nodes",
+"crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:EPSG::3857" } },
+"features": [
+{ "type": "Feature", "properties": { "id": 0 }, "geometry": { "type": "Point", "coordinates": [ -4.0, -9.35 ] } },
```

Comment:
None of the coordinates have the frame. Generally isn't required but for the best practices example it should have them